source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -567,8 +567,11 @@ AdaptiveNamespaceStart
         [propget] HRESULT FallbackText([out, retval] HSTRING* value);
         [propput] HRESULT FallbackText([in] HSTRING value);
 
-        [propget] HRESULT BackgroundImage([out, retval] Windows.Foundation.Uri** value);
-        [propput] HRESULT BackgroundImage([in] Windows.Foundation.Uri* value);
+        [propget] HRESULT BackgroundImageUri([out, retval] HSTRING* value);
+        [propput] HRESULT BackgroundImageUri([in] HSTRING value);
+
+        [propget] HRESULT IsBackgroundImageUriRelative([out, retval] boolean* value);
+        [propput] HRESULT IsBackgroundImageUriRelative([in] boolean value);
```

Comment:
If you haven't already, run this by Matt to make sure he's okay with both the breaking change and the loss of a strong type.